dev-tools/reconfigurator-cli/tests/output/cmds-example-stdout

```diff
@@ -553,22 +553,12 @@ T ENA ID                                   PARENT
 > blueprint-plan ade5749d-bdf3-4fab-a8ae-00bea01b3a5a
 INFO skipping noop image source check for all sleds, reason: no target release is currently set
 WARN cannot issue more MGS-driven updates (no current artifacts)
-INFO some zones not yet up-to-date, sled_id: 89d02b1b-478c-401a-8e28-7a26f74fa41b, zones_currently_updating: [ZoneCurrentlyUpdating { zone_id: b3c9c041-d2f0-4767-bdaf-0e52e9d7a013 (service), zone_kind: InternalNtp, reason: MissingInInventory { bp_image_source: InstallDataset } }]
-generated blueprint 86db3308-f817-4626-8838-4085949a6a41 based on parent blueprint ade5749d-bdf3-4fab-a8ae-00bea01b3a5a
-planning report for blueprint 86db3308-f817-4626-8838-4085949a6a41:
-* discretionary zone placement waiting for NTP zones on sleds: 89d02b1b-478c-401a-8e28-7a26f74fa41b
-* missing NTP zone on sled 89d02b1b-478c-401a-8e28-7a26f74fa41b
-* only placed 0/1 desired clickhouse zones
-* only placed 0/3 desired crucible_pantry zones
-* only placed 0/3 desired internal_dns zones
-* only placed 0/3 desired nexus zones
-
+error: generating blueprint: no active Nexus zones exist in parent blueprint
```

Comment:
This is a result of using the `--no-zones` argument to the reconfigurator-cli -- which now makes less sense, because it results in a planning error.

Reply:
Turns out, removing `--no-zones` can work fine. It definitely makes the output noisier, but the tests seem to still work. Did that in https://github.com/oxidecomputer/omicron/pull/8936/commits/ff13b718f3735a84251fdb34eba79b602cdf513f